pbf/views.py

```diff
@@ -84,12 +93,12 @@ def new_game(request):
         'Voice': 0,
         'Roots': 1,
         'Gaze': 1,
-        'Vigil': 1,
+        'Vigil': 0,
         'Behemoth': 0,
         'Earthquakes': 1,
         'Breath': 1,
-        'Waters': 4,
-        'Violence':1,
+        'Waters': 0,
+        'Violence': 2,
```

Comment:
I make this comment as a personal preference only, as I have no authority on whether this pull request gets accepted or not:

I think it makes sense to remove this `Violence:` line altogether. There is currently no aspect that changes the tracks, so there is currently not a need to define any aspect as having a different `base_energy_gain` than the spirit it modifies. Thus the conditional `if aspect in spirit_base_energy_gain.keys():` could potentially be removed as well. It's possible you'd consider keeping it in there for future-proofing, but I think it's easy enough to add it when it becomes relevant.

Reply:
Good call! I lean towards removing the unnecessary Violence aspect from the list, and that seems to match precedent, while leaving the aspect check in place for both energy dictionaries.